lib/Rex/Service/OpenBSD.pm

```diff
@@ -24,34 +24,17 @@ sub new {
   bless( $self, $proto );
 
   $self->{commands} = {
-    start   => '/etc/rc.d/%s start',
-    restart => '/etc/rc.d/%s restart',
-    stop    => '/etc/rc.d/%s stop',
-    reload  => '/etc/rc.d/%s reload',
-    status  => '/etc/rc.d/%s status',
-    action  => '/etc/rc.d/%s %s',
+    start        => '/usr/sbin/rcctl start %s',
+    restart      => '/usr/sbin/rcctl restart %s',
+    stop         => '/usr/sbin/rcctl stop %s',
+    reload       => '/usr/sbin/rcctl reload %s',
+    status       => '/usr/sbin/rcctl check %s',
```

Comment:
Well done preserving backwards compatibility! Do you think it would be a good idea to also add an explicit `check => ...` line for this?

Since I'm not an active OpenBSD user, I wonder if that would be handy for OpenBSD users or not. I mean to be able to use the same "words" as they would use on the command line.